.github/workflows/build-envoy-images-release.yaml

```diff
@@ -188,12 +191,12 @@ jobs:
           build-args: |
             BUILDER_BASE=quay.io/${{ github.repository_owner }}/cilium-envoy-builder:${{ env.BAZEL_VERSION }}-${{ env.BUILDER_DOCKER_HASH }}
             BAZEL_BUILD_OPTS=--remote_upload_local_results=false
-            ARCHIVE_IMAGE=quay.io/${{ github.repository_owner }}/cilium-envoy-builder:master-archive-latest
+            ARCHIVE_IMAGE=quay.io/${{ github.repository_owner }}/cilium-envoy-builder:${{ env.ENVOY_MINOR_RELEASE }}-archive-latest
```

Comment:
Now that the Envoy version and brach name may need to be different, how about using the (release) branch name here instead? So use `v1.23` here, but derive it from the branch name rather than from the Envoy version? That way we could use the same script (copied or otherwise) for multiple release branches even if they happened to be on the same Envoy version in future?

Reply:
ah good point :100: